content/docs/reference/pkg/gitlab/_index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "gitlab"
```

Comment:
Looks like we need to add gitlab to the lookup map. I'll add it in a PR I have going for something else.